docs/Design/Numbers/Native Math.md

```diff
@@ -22,8 +22,20 @@ variable : bar int -7
 
 math absolute : bar
 ```
+### `math floor`
 
+`math floor` `:` `name`
 
+Generates code to compute the previous largest integer of the given number.
```

Comment:
`previous large integer` is a little unclear. I'm partial to MDN's definition that it gives the largest integer <= the number.